packages/ember-views/lib/compat/has-block.js

```diff
@@ -0,0 +1,5 @@
+import { symbol } from 'ember-metal/utils';
```

Comment:
This doesn't seem like a compat type thing to me, maybe we can move it to `packages/ember-views/lib/system/` instead?